### ISSUE TITLE
IpException `configure` method throws Exception

### ISSUE THREAD

**Issue by fabiopellegrini** [Respect/Validation#1012]
Executing
```
$e = new Respect\Validation\Exceptions\IpException();
$e->configure('some-name');
```

Will cause the following error:
`Undefined index: networkRange`

In IpException.php 
```
  public function configure($name, array $params = [])
    {
        if ($params['networkRange']) {
            $range = $params['networkRange'];
    . . .
```
Should be:
```
  public function configure($name, array $params = [])
    {
        if (isset($params['networkRange'])) {
            $range = $params['networkRange'];
```

**Comment by henriquemoody** [comment]
Thanks for reporting! Can you also explain why are you creating instances of that exception?

**Comment by henriquemoody** [comment]
I will be released in the version 1.1.17.

Thanks for reporting! 🐼

**Comment by fabiopellegrini** [comment]
Hi @henriquemoody , thank you for the fix! :)
I created an object which uses Respect validation and I wanted to automatically set the `name` parameter of the exceptionts raised by the validator.
When I started writing tests I realized that IpException had a this little bug in `configure` method.
